### DISCUSSION TITLE
Virus software detects 0.4.5 update!

### DISCUSSION THREAD

**Discussion by tylere661** [OpenRCT2/OpenRCT2#20201] [Q&A]
Hello, I have tried to download the latest Openrct2 update 0.4.5, but 2 virus software programs found a virus in the download.  Both AVG virus software and the total virus online scanner.  Any suggestions?

**Comment by duncanspumpkin** [comment]
Assuming you've downloaded it from openrct2.io you'll just have to mark it as okay and tell the virus software it's wrong. Eventually after enough people tell the virus software that it's wrong they will mark it as good.

**Reply by tylere661** [reply]
Thank you very much!
